### PR TITLE
feat(store): Adds AM3 to microcenter

### PR DESCRIPTION
**Pull request by AndyGOBrien**
### Description

This just adds AM3 processors to the microcenter store.

### CODE REVIEW

File: src/store/model/microcenter.ts [hunk 1]
```diff
@@ -89,6 +89,30 @@ for (const microCenterLocation of microCenterLocations) {
...
+				brand: 'amd',
+				model: '5800x',
+				series: 'ryzen5800',
+				url: `https://www.microcenter.com/product/630284/amd-ryzen-7-5800x-vermeer-38ghz-8-core-am4-boxed-processor/?storeid=${storeId}`
+			},
+			{
+				brand: 'amd',
+				model: '5900x',
+				series: 'ryzen5900',
+				url: `microcenter.com/product/630283/amd-ryzen-9-5900x-vermeer-37ghz-12-core-am4-boxed-processor/?storeid=${storeId}`
```
Comment (jef) [inline src/store/model/microcenter.ts:109]:
```suggestion
				url: `https://www.microcenter.com/product/630283/amd-ryzen-9-5900x-vermeer-37ghz-12-core-am4-boxed-processor/?storeid=${storeId}`
```

**Review by jef — CHANGES_REQUESTED** [review]

**Comment by jef** [comment]
Looks good otherwise; Thanks!

**Comment by AndyGOBrien** [comment]
@jef Just bumping this, can I get your signature real quick?

**Review by jef — APPROVED** [review]
Looks good to me! Thanks!

**Pull request merged**